### DISCUSSION TITLE
Month/year navigation and arrow

### DISCUSSION THREAD

**Discussion by Piero87** [gpbl/react-day-picker#1630] [Ideas]
I want to know if it's possible to have month/year navigation and arrow month navigation at the same time
![Screenshot 2023-01-01 alle 22 51 38](https://user-images.githubusercontent.com/768952/210185667-2e5fd741-a8a2-4cb1-808a-ca28d837fe05.png)
![Screenshot 2023-01-01 alle 22 51 59](https://user-images.githubusercontent.com/768952/210185668-daebf29c-f31e-48d2-92d1-4aa3a2458540.png)

I have activated the dropdown with this captionLayout="dropdown" but I don't se arrows anymore

**Comment by gpbl** [comment]
This is being requested often! The issue I had when building it was purely aesthetic - I could not accommodate the text when long enough in the horizontal space. Open to ideas and contributions here!

**Reply by seanockert** [reply]
Hi gpbl, I've added a PR for this here: https://github.com/gpbl/react-day-picker/pull/1678
Horizontal space doesn't seem to be an issue with the default styles.
